integration/scripts/src/runTest.ts

```diff
@@ -0,0 +1,76 @@
+import { ChainlinkNode } from './chainlinkNode'
+import { fetchTests } from './tests'
+import { fetchArgs, fetchCredentials } from './common'
+import * as assert from './asserts'
+
+const main = async () => {
```

Comment:
Why not use a test framework like Jest here?

Reply:
The reason for this is that it seemed easier to just build a small framework to allow for very specific testing against a Chainlink node, as opposed to pulling in an external library and then trying to make that fit with this use-case. The testing library here is only about 100 LOC anyways. For now it seems to me like this is the simplest approach, but if you can think of any reasons why Jest can be better here I'm definitely open to that.